cli/crates/backend/src/api/login.rs

```diff
@@ -86,8 +81,13 @@ pub async fn login(message_sender: MspcSender<LoginMessage>) -> Result<(), ApiEr
         Err(error) => return Err(ApiError::ReadUserDotGrafbaseFolder(error)),
     }
 
-    let port = find_available_port_in_range(EPHEMERAL_PORT_RANGE, LocalAddressType::Localhost)
-        .ok_or(ApiError::FindAvailablePort)?;
+    let listener = TcpListener::bind((Ipv4Addr::LOCALHOST, 0))
+        .await
+        .map_err(|_| ApiError::FindAvailablePort)?;
```

Comment:
OPT: could also probably have been done with something like `.ok().and_then(...).ok_or_else(|| ApiError::FindAvailablePort)?` chain I think

Reply:
Will take a look into this for a future version